rakelib/artifacts.rake

```diff
@@ -214,7 +214,7 @@ namespace "artifact" do
 
     #without JDKs
     @bundles_jdk = false
-    system("./gradlew bootstrap") #force the build of Logstash jars
+    raise "Gradle failed to execute deleteLocalJdk bootstrap" unless system("./gradlew bootstrap") #force the build of Logstash jars
```

Comment:
There is a typo here, but I wonder if it makes sense to factor this out into a function to avoid the possibility of typos, and to avoid the repeated `raise...unless` pattern to improve readability - something like:

```
def safe_system(*args)
    if !system(*args)
      status = $?
      raise "Got exit status #{status.exitstatus} attempting to execute #{args.inspect}!"
    end
  end
```

which would make this pattern: 
```suggestion
safe_system("./gradlew bootstrap") #force the build of Logstash jars
```

Reply:
Very interesting suggestion, thanks for it :-)